docs/home/contribute/participating.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Participating in SIG-DOCS
+---
+
+{% capture overview %}
+
+SIG-DOCS is one of the [special interest groups](https://github.com/kubernetes/community/blob/master/sig-list.md) within the Kubernetes project, focused on writing, updating, and maintaining the documentation for Kubernetes as a whole.
+
+{% endcapture %}
+
+{% capture body %}
+
+SIG Docs welcomes content and reviews from all contributors. Anyone can open a pull request (PR), and anyone is welcome to comment on content or pull requests in progress.
+
+Within the Kubernetes project, you may also become a member, reviewer, or approver.
+These roles confer additional privileges and responsibilities when it comes to approving and committing changes.
+See [community-membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for more information on how membership works within the Kubernetes community.
+
+## Roles and Responsibilities
+
+- Members
+
+Any member of the [Kubernetes organization](https://github.com/kubernetes) can review a pull request, and SIG Docs team members frequently request reviews from other SIG members for technical accuracy. 
+You can indicate your approval by adding a comment of `/lgtm` to a pull request.
+
+- Reviewers
+
+Reviewers are individuals who review documentation pull requests. You can become a reviewer by adding your github handle to the [OWNERS file](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md) within the [website repo](https://github.com/kubernetes/website).
+(Additions to the OWNERS file require contributors to meet certain requirements. See the [Kubernetes community repo](https://github.com/kubernetes/community/blob/master/community-membership.md) for more information.
+
+Our automation assigns reviewers to pull requests, and contributors can request your review with a comment on the pull request: `/assign [your_github_handle]`. To indicate that a pull request requires no further changes, a reviewer adds a comment to the pull request `/lgtm`.
+
+Approvers can indicate their approval with a comment: `/approve`.
+
+Reviewers and approvers can add a `/hold` comment to prevent the pull request from being merged. Another reviewer or approver can remove a hold with the comment: `/hold cancel`.
+
+When both of a `lgtm` and `approval` tag have been added, and there are no `hold` tags, the automation on the repository will merge the pull request.
+
+**Note:** There is a special case when an approver uses the comment: `/lgtm`. In these cases, the automation will add both `lgtm` and `approve` tags, skipping any further review.
```

Comment:
Since the approver is a special case that result in a /approve being automatically added. It would be good to provide a short cheat sheet that describes the proper way for an approver to handle the key use cases. For example:
If the approver would like a tech review done before the PR can be merged, he or she should do the following:
1.  Use /hold to denote this PR should be merged yet.
2.  Add a comment requesting that a tech review be performed.
3.  Inform the tech reviewer by .... (doing an assign?)
4.  Once a tech review has been performed a /hold cancel should be added by the approver

If no tech review is required and all that is needed is an /lgtm then no special bot commands are required.